core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -103,10 +102,10 @@ func (eb *ethBroadcaster) Start() error {
 }
 
 func (eb *ethBroadcaster) Close() error {
-	if !eb.OkayToStop() {
-		return errors.New("EthBroadcaster is already stopped")
-	}
+	return eb.StopOnce("eth broadcaster", eb.close)
+}
```

Comment:
hm here as well, maybe better to have it unified in all services in a single Start/Close instead of two methods

Reply:
I wasn't sure: I kept separate methods if it was a long method, and I merged them into a single one if it was just a few lines. The `fn` closure adds another level of indentation

Reply:
hm true - i guess i don't mind either approach, but i think would be good to keep just one of them, consistent in all services

Reply:
I removed the extra methods